packages/kmod-5.10-nvidia/kmod-5.10-nvidia.spec

```
@@ -84,7 +84,8 @@ install -d %{buildroot}%{_cross_unitdir}
 install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/{drivers,ld.so.conf.d}
 
 KERNEL_VERSION=$(cat %{kernel_sources}/include/config/kernel.release)
-sed -e "s|__KERNEL_VERSION__|${KERNEL_VERSION}|" %{S:200} > nvidia.conf
+sed -e "s|__KERNEL_VERSION__|${KERNEL_VERSION}|" %{S:200} | sed -e "s|__PREFIX__|%{_cross_prefix}|" \
```

nit: you should be able to combine both edits in one command:
```suggestion
sed \
  -e "s|__KERNEL_VERSION__|${KERNEL_VERSION}|" %{S:200} \
  -e "s|__PREFIX__|%{_cross_prefix}|" \
```